extra-files/dependency_resolvers_conf.xml

```diff
@@ -0,0 +1,28 @@
+<dependency_resolvers>
```

Comment:
I wouldn't change the default resolver order, this doesn't help anything and may be very confusing when you have installed a toolshed dependency and it uses the -- eventually -- wrong conda dependency. Think for example velvet compiled with a different max k-mer.